sirepo/package_data/template/radia/geometry.py.jinja

```diff
@@ -4,6 +4,7 @@ import h5py
 import os
 import re
 from pykern.pkcollections import PKDict
+import sirepo.mpi
```

Comment:
radia_tk name (below) is confusing to me. I don't think this has anything to do with "tk", and it should be named differently from other modules (jupyter).

Reply:
Was originally a direct copy from ```jupyter-rs-radia``` with an eye toward a single library for both.  I can't remember why I preferred a "toolkit" to "utilities".  So is the idea to rename to radia_util and put the new stuff in there as well?

Reply:
a537825

Reply:
"tk" didn't translate to "toolkit" in my mind. "util" maps to "uti" in radia world, which is why I thought of it. We also use util other places in sirepo. I think I was confused with VTK and radias-vtk. Anyway, thanks for changing to radia_util. :)